tests/step_implementers/create_container_image/test_buildah_create_container_image.py

```diff
@@ -18,15 +19,17 @@ def create_step_implementer(
             step_config={},
             step_name='',
             implementer='',
-            workflow_result=None,
+            results_dir_path='',
+            results_file_name='',
             work_dir_path=''
     ):
         return self.create_given_step_implementer(
             step_implementer=Buildah,
             step_config=step_config,
             step_name=step_name,
             implementer=implementer,
-            workflow_result=workflow_result,
```

Comment:
why this change? this looks like a revert of a change that went in a few days ago.

Reply:
i am guessing this is because this change was started prior to the workflow result changes and function signature for step implementers.

see the "new way" here: https://github.com/ploigos/ploigos-step-runner/blob/main/tests/step_implementers/sign_container_image/test_podman_sign.py#L75-L90

Reply:
@blakerollins this still needs to be cleaned up